Lesson_12/l12_h01_recursive_function.py

```diff
@@ -0,0 +1,29 @@
+"""
+HW:12[Volkov Anton].
+
+Given list of list of list etc of integers
+write recursive function that will accept as argument target list
+and return sum of all integers inside it
+Input: [[[[1, 4, 5], [[6, 9],[[[8, 1], 7], 3], 2], 7], 5, 2], 9, [1, 2]]
+Output: Target sum = 72
+"""
+target_list = [[[[1, 4, 5], [[6, 9], [[[8, 1],
+               7], 3], 2], 7], 5, 2], 9, [1, 2]]
+
+
+def get_list_sum(digits_list, sum_of_numbers=0):
+    """Get the sum of all elements from digits_list using recursion."""
+    for i in digits_list:
```

Comment:
```suggestion
    for el in digits_list:
```
"i" usually means simple number - thus better to use other naming for item or list element